components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/constant/OrganizationManagementConstants.java

```diff
@@ -347,6 +347,11 @@ public enum ErrorMessages {
                 "cannot be empty."),
         ERROR_CODE_MANAGED_ORGANIZATION_CLAIM_UPDATE_NOT_ALLOWED("60089", "The managed organization " +
                 "is not allowed to modify.", "The managed organization is a read only property."),
+        ERROR_CODE_EMAIL_DOMAIN_ASSOCIATED_WITH_DIFFERENT_ORGANIZATION("60090", "The user is not allowed " +
+                "to register with this organization.", "The email domain provided has been associated with a " +
+                "different organization."),
+        ERROR_CODE_EMAIL_DOMAIN_NOT_MAPPED_TO_ORGANIZATION("60091", "The user is not allowed to register with " +
```

Comment:
Isnt ERROR_CODE_EMAIL_DOMAIN_NOT_MAPPED_TO_ORGANIZATION this enough?
What is the scenario we need to show that email domain is associated with different org to end user who trying to register?

Reply:
60090 - this is for scenarios where the organization doesn't have any mapped domains but tries to register a user with an email domain that is already associated for another organization

60091 - this is for scenarios where the organization already has a mapped domain but tries to register user with a different domain